src/ui/settings/ViewTipsAndTricks/TipsAndTricksSection.qml

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import components 0.1
+
+ColumnLayout {
+    id: tipsAndTricksSection
+
+    property var flickableParent
+
+    property string title
+    property string description
+
+    property bool hasGuides
+    property var guidesModel
+
+    property bool hasTutorials
+    property var tutorialsModel
+
+    visible: hasGuides || hasTutorials
+
+    // Title
+    VPNBoldLabel {
+        Layout.topMargin: 32
```

Comment:
nit: I think moving `Layout.topMargin: 32` to where we instantiate the TipsAndTricksSection's in ViewTipsAndTricks makes more sense from a readability perspective, and also, we don't have to dig down to the component level and make special exceptions if we ever want to change the spacing between sections